extensions/wikia/CreateNewWiki/tests/tasks/CreateDatabaseTest.php

```diff
@@ -120,7 +120,7 @@ public function testPrepareShouldTakeOtherDbNameWhenWikiCitiesTaken( $taskContex
 		$result = $task->prepare();
 
 		//then
-		$taskContextData = $taskContext->getAllProperties();
+		$taskContextData = $taskContext->getLoggerContext();
 		$this->assertEquals( true, $result->isOk());
 		//We expect that the db name will have the original name + some suffix i.e. textWiki123
 		$this->assertNotEquals( $taskContextExpected[ 'dbName' ], $taskContextData[ 'dbName' ] );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "dbName" 11 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)